src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  padding: $calendar-padding;
+  gap: $gap-size;
+
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    height: $day-size;
+    width: $day-size;
+    border: $border-size solid black;
+    background-color: #eee;
+    transition: all 0.5s;
```

Comment:
it's not a good practice to set `all` as transition-property. Specify what rules exactly you'd like to change